presto-main/src/main/java/com/facebook/presto/server/ServerMainModule.java

```diff
@@ -348,10 +349,11 @@ protected void setup(Binder binder)
         binder.bind(Metadata.class).to(MetadataManager.class).in(Scopes.SINGLETON);
 
         // statistics calculator
+        binder.bind(CostComparator.class).in(Scopes.SINGLETON);
         binder.bind(StatsCalculator.class).to(CoefficientBasedStatsCalculator.class).in(Scopes.SINGLETON);
         binder.bind(CostCalculator.class).to(CostCalculatorUsingExchanges.class).in(Scopes.SINGLETON);
         binder.bind(CostCalculator.class)
-                .annotatedWith(Names.named("estimatedExchangesCostCalculator"))
+                .annotatedWith(EstimatedExchanges.class)
```

Comment:
This should be squashed with the commit that introduced `.annotatedWith(Names.named("estimatedExchangesCostCalculator"))`